backend/resolvers.js

```diff
@@ -1,45 +1,139 @@
 const {AuthenticationError} = require("apollo-server-errors");
-
 const uuidv1 = require('uuid/v1');
-const items = require('./data.js');
-const users = require('./users.js');
 const {createJWTToken} = require('./helper/jwt.js');
 
 // Resolvers define the technique for fetching the types defined in the
-// schema. This resolver retrieves items from the "items" array above.
+// schema. This resolver retrieves todos from the neo4j db
 const resolvers = {
     Query: {
-        items: () => items
+        todos: async (parent, args, context) => {
+            const currentUser = context.user.name;
+            let page = 0;
+            let size = 20;
+            if (typeof args.page != "undefined") page = args.page;
+            if (typeof args.size != "undefined") size = args.size;
+
+            console.log(`INFO - Got 'ALL_TODOS_QUERY' from user ‘${currentUser}'`);
+            const session = context.driver.session();
+            try {
+                const todosQuery = await session.run(
+                    'MATCH (t:Todo)-[:BELONGS]->(u:User)\n' +
+                    'WHERE u.name = $userName\n' +
+                    'RETURN t, u\n' +
```

Comment:
```suggestion
                    'RETURN t {.*, user: properties(u) }' +
```
That way you can skip the calls of `.properties` down below :point_down: